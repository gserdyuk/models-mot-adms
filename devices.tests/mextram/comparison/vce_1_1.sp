* gnucap mextram test vce

.include my-mextram.spinc

VB B 0 dc 1.0
VC C 0 dc 3.0
VS S 0 dc 0.0
X1 C B 0 S dt mextram_ckt

.dc vb 0 2 0.1 
.print dc  v(b) v(c) v(b) v(s) i(VC) i(vb) i(Vs) 
 
.end
