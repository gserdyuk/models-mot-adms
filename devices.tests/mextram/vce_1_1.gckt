* gnucap mextram test vce

.include my-mextram.inc

VB B 0 dc 1.0
VC C 0 dc 3.0
VS S 0 dc 0.0
X1 C B 0 S dt mextram_ckt

.print dc  v(vb) v(c) v(b) v(s) i(VC) i(Vb) i(Vs) 
.dc vb 0 2 0.1   basic
 
.end
