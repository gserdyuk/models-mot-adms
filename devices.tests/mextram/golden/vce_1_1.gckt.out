Gnucap 2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
Q: already installed, replacing
stashing as Q:0
* gnucap mextram test vce 
#           v(VB)      v(c)       v(b)       v(s)       i(VC)      i(VB)      i(VS)     
 0.         30.E-18    3.         0.         0.         0.         3.E-12     0.        
 0.1        0.1        3.         0.1        0.         0.         0.         0.        
 0.2        0.2        3.         0.2        0.         0.         0.         0.        
 0.3        0.3        3.         0.3        0.         0.         0.         0.        
 0.4        0.4        3.         0.4        0.         0.         0.         0.        
 0.5        0.5        3.         0.5        0.         0.         0.         0.        
 0.6        0.6        3.         0.6        0.        -228.74E-9  0.         0.        
 0.7        0.7        3.         0.7        0.        -10.461E-6 -71.465E-9  0.        
 0.8        0.8        3.         0.8        0.        -441.25E-6 -3.1897E-6  0.        
 0.9        0.9        3.         0.9        0.        -0.0064722 -57.292E-6  0.        
 1.         1.         3.         1.         0.        -0.01863   -299.98E-6  0.        
 1.1        1.1        3.         1.1        0.        -0.031382  -884.27E-6  0.        
 1.2        1.2        3.         1.2        0.        -0.043192  -0.0018495  0.        
 1.3        1.3        3.         1.3        0.        -0.053693  -0.0031404  0.        
 1.4        1.4        3.         1.4        0.        -0.062923  -0.0046856  6.E-21    
 1.5        1.5        3.         1.5        0.        -0.071035  -0.0064264  233.1E-18 
 1.6        1.6        3.         1.6        0.        -0.078187  -0.0083192  4.468E-12 
 1.7        1.7        3.         1.7        435.E-15  -0.084511  -0.010333   43.524E-9 
 1.8        1.8        3.         1.8        1.1885E-9 -0.08992   -0.012595   118.85E-6 
 1.9        1.9        3.         1.9        20.57E-9  -0.092212  -0.017887   0.002057  
 2.         2.         3.         2.         50.578E-9 -0.091127  -0.026234   0.0050578 
